js/main.js

```diff
@@ -108,7 +153,12 @@ function toggleNoEntries(event) {
     $noEntryMessage.classList.remove('hidden');
   }
 }
-
+// function to reset title to be utilized inside submitInfo function
+function updateTitleToNewEntry() {
+  const entryHeader = document.querySelector('.new-entry-header');
+  entryHeader.innerHTML = 'New Entry';
```

Comment:
You shouldn't use `innerHTML`, it's a security concern. You should use `textContent`